src/public/modules/forms/admin/directiveViews/settings-form.client.view.html

```diff
@@ -283,7 +283,7 @@
                     type="text"
                     name="esrvcId"
                     ng-model="tempForm.esrvcId"
-                    placeholder="Enter your e-service ID here"
+                    placeholder="{{ ['SP', 'MyInfo'].includes(type.val) && 'Enter Singpass e-service ID' || 'Enter Corppass e-service ID' }}"
```

Comment:
a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) would probably be more appropriate for this case